.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -2,7 +2,7 @@
 name: 🐛 Bug report
 about: Create a report to help us improve
 title: "[BUG]"
-labels: 'bug, untriaged, Beta'
+labels: 'bug, untriaged'
 assignees: ''
 ---
 
```

Comment:
Hi dB, I noticed you created a repository called `.gtihub`, I think the issue template there can be applied to all repositories in the whole organization (https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file), so are we going to remove the issue template in each repository, and use the uniform template instead?

Reply:
I don't believe that's how it works actually, does it?

Reply:
I think you are correct. https://github.community/t/issue-template-for-organization-does-not-work/2049/7

Reply:
However, I couldn't prove it actually works for us, this feature might have its limitation. Let's have a try at some time. 😉